WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -10,6 +12,8 @@ final class PaymentCaptureOrchestrator {
 
     private let celebration = PaymentCaptureCelebration()
 
+    private var walletSuppressionRequestToken = PKSuppressionRequestToken(0)
```

Comment:
Is there any guarantee that `0` won't ever be returned? An optional `PKSuppressionRequestToken` might be safer here.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4218/commits/784c6cb0cefd01a19e1aa6665d837b985f499c22